modules/@apostrophecms/rich-text-widget/ui/apos/tiptap-extensions/Link.js

```diff
@@ -1,10 +1,13 @@
 import Link from '@tiptap/extension-link';
 export default (options) => {
   return Link.extend({
-    defaultOptions: {
-      openOnClick: false,
-      linkOnPaste: true,
-      HTMLAttributes: {}
+    addOptions() {
+      return {
+        ...this.parent?.(),
+        openOnClick: false,
+        linkOnPaste: true,
+        HTMLAttributes: {}
```

Comment:
TODO: find how to override these options (in core rich text widget in apostrophe, on project side, and at area level), now that we're adding options correctly here, it could lead to some complaints (e.g `openOnClick` is now well set to false, so users might want to set it back to true).
cc @boutell